src/types.ts

```diff
@@ -189,11 +195,11 @@ export namespace OfCoerce {
       > // Literal
         ? SchemaPair<Shape>
         : Utils.Debrand<Shape> extends boolean // Boolean
-        ? SchemaPair<Shape, BooleanConstructor>
+        ? SchemaPair<Shape, BooleanConstructor | To<Shape>>
         : Utils.Debrand<Shape> extends string // String
-        ? SchemaPair<Shape, StringConstructor>
+        ? SchemaPair<Shape, StringConstructor | To<Shape>>
         : Utils.Debrand<Shape> extends number // Number
-        ? SchemaPair<Shape, NumberConstructor>
+        ? SchemaPair<Shape, NumberConstructor | To<Shape>>
```

Comment:
To allows primitives and constructors

Reply:
I love it! Adding primitives was the next step! I'll make sure to add the runtime for it.